src/scripts/import-sdr-shipments-sheet/add-countries.ts

```diff
@@ -75,41 +69,19 @@ export async function addCountries(shipments: ShipmentCsv[]) {
   const validCountries: Country[] = [
     ...resultsMap[UploadWorkflowStatus.SUCCESS],
     ...resultsMap[UploadWorkflowStatus.ALREADY_EXISTS],
-  ].reduce((countries: Country[], workflow: CountryUploadWorkflow) => {
-    return [...countries, workflow.data];
-  }, [] as Country[]);
+  ].map((workflow) => workflow.data);
 
   return validCountries;
 }
 
-const isFulfilled = <T>(
-  value: PromiseSettledResult<T>,
-): value is PromiseFulfilledResult<T> => {
-  return value.status === "fulfilled";
-};
-
-const _isRejected = <T>(
-  value: PromiseSettledResult<T>,
-): value is PromiseRejectedResult => {
-  return value.status === "rejected";
-};
-
 /* Consolidate Countries
  * ------------------------------------------------------ */
-function consolidateCountries(shipments: ShipmentCsv[]) {
-  const countries = shipments.reduce((countries: Set<string>, shipment) => {
-    if (shipment.sendingCountry !== "") {
-      countries.add(shipment.sendingCountry);
-    }
-
-    if (shipment.receivingCountry !== "") {
-      countries.add(shipment.receivingCountry);
-    }
-
-    return countries;
-  }, new Set<string>());
-
-  return countries;
+function consolidateCountries(shipments: ShipmentCsv[]): Set<string> {
+  return new Set(
+    shipments.flatMap((s) =>
+      [s.sendingCountry, s.receivingCountry].filter((v) => v !== ""),
+    ),
```

Comment:
@corasaurus-hex

**question:** Should we set some variable naming conventions?  Personally I prefer to avoid shorthand variable names (clarity for other devs, code searchability, etc).  For example in this case I'd use `shipment` or `country` instead of `s` and `v`.  But I don't really care as long as we're consistent in how we approach it.

Reply:
I usually scale the length of my variable names with the complexity and length of the function, so for simple and smaller functions I use smaller variables because it's trivial to see where the value comes from. Does that seem fair? It's a subtle thing

Reply:
@corasaurus-hex that certainly sounds reasonable to me!  Do you think this would be a good one to kick back to the wider team on slack to see if anyone has strong preferences?  The we can note it as a convention in the readme or something, which will help folks who are new to our codebase (Hacktoberfest, etc).

Reply:
A style guide?

Reply:
I'm reluctant to be *very* prescriptive on styles, given how subjective style is, and less prescriptive when it comes to folks with less experience, but we could certainly add a style guide as a task leading up to hacktoberfest so that folks can have some guidance.

I wonder how many of these things could be made linter rules, too?